src/testData.ts

```diff
@@ -0,0 +1,49 @@
+
+import * as Types from './types'
+
+export const sectionData: Types.sectionData = {
+  name: 'Intro',
+  description: 'This intro section consists of a tuba quartet in the style of DJ Templeton & The Windsurfers',
+  numRevisions: 42,
+};
+
+export const currentChordProgression: Types.chordProgression = ['C', 'Dm', 'F', 'G']
+
+export const files: Types.file[] = [
```

Comment:
I think I would actually expect `Types` to be written as `types`. Namespaces like that are typically lowercased